lib/src/widget/built_in/built_in.dart

```diff
@@ -63,29 +63,20 @@ class BuiltInContent extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    Widget content = Text(
-      title,
-      style: style.titleTextStyle(context)?.copyWith(
-            color: foregroundColor,
-          ),
-    );
-
     final showColumn =
         description?.isNotEmpty == true || showProgressBar == true;
 
     if (showColumn) {
-      content = Column(
+      return Column(
         mainAxisSize: MainAxisSize.min,
         crossAxisAlignment: CrossAxisAlignment.start,
         children: [
-          content,
+          buildTitle(context),
           if (description?.isNotEmpty == true) ...[
             const SizedBox(height: 4),
             Text(
               description!,
-              style: style.descriptionTextStyle(context)?.copyWith(
-                    color: foregroundColor?.withOpacity(.7),
-                  ),
+              style: style.descriptionTextStyle(context),
```

Comment:
We have to use the `foregroundColor` in both title and description.

Reply:
The problem is if I add a `descriptionTextStyle` in my `style` you are forcing me to have `foregroundColor?.withOpacity(.7)`.

check out line 82 of `filled_style.dart`:
```
color: foregroundColor(context).withOpacity(.9), 
```
This won't apply for example as far as you are over writing it in `built_in.dart`
So this is not intuitive.